src/main/java/org/itsallcode/openfasttrace/importer/legacytag/LegacyTagImporterFactory.java

```diff
@@ -39,43 +40,61 @@
 
     private static LegacyTagImporterConfig defaultConfig = new LegacyTagImporterConfig();
 
-    private final List<PathConfig> pathConfigs;
+    private final Supplier<LegacyTagImporterConfig> config;
 
     public LegacyTagImporterFactory()
     {
-        this(defaultConfig.getPathConfigs());
+        this(() -> defaultConfig);
     }
 
-    public LegacyTagImporterFactory(final List<PathConfig> pathConfigs)
+    public LegacyTagImporterFactory(final LegacyTagImporterConfig config)
     {
-        this.pathConfigs = pathConfigs;
+        this(() -> config);
+    }
+
+    public LegacyTagImporterFactory(final Supplier<LegacyTagImporterConfig> config)
+    {
+        this.config = config;
     }
 
     @Override
-    public boolean supportsFile(final Path file)
+    public boolean supportsFile(final Path path)
     {
-        return findConfig(file).isPresent();
+        return findConfig(path).isPresent();
     }
 
     private Optional<PathConfig> findConfig(final Path file)
```

Comment:
You changed the parameter to "path" in the calling function but kept is as "file" here.
Suggest using "path" uniformly.

Reply:
✔️